pwwka.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency("rspec")
   s.add_development_dependency("resque")
   s.add_development_dependency("resque-retry")
+  s.add_development_dependency("sidekiq")
```

Comment:
Is there any concern/downside to including sidekiq in resque projects?  I guess this already happens for the inverse, resque being included in sidekiq projects?

Reply:
I think it's OK, since Resque doesn't do anything just be being included.  It should stay inert if the code doens't use it.

Reply:
I don't think so, since [the Sidekiq](https://github.com/mperham/sidekiq#sidekiq) docs say "You can have Sidekiq and Resque run side-by-side at the same time".

Reply:
also [the Sidekiq docs](https://github.com/mperham/sidekiq#sidekiq) say "You can have Sidekiq and Resque run side-by-side at the same time." so I think it should be okay.